test/int/databaseuser_unprotected_test.go

```diff
@@ -785,40 +795,30 @@ func checkSecretsDontExist(namespace string, secretNames []string) func() bool {
 
 func checkUserInAtlas(projectID string, user mdbv1.AtlasDatabaseUser) {
 	By("Verifying Database User state in Atlas", func() {
-		atlasDBUser, _, err := atlasClient.DatabaseUsers.Get(context.Background(), user.Spec.DatabaseName, projectID, user.Spec.Username)
+		atlasDBUser, _, err := atlasClient.DatabaseUsersApi.
+			GetDatabaseUser(context.Background(), projectID, user.Spec.DatabaseName, user.Spec.Username).
+			Execute()
 		Expect(err).ToNot(HaveOccurred())
-		operatorDBUser, err := user.ToAtlas(context.Background(), k8sClient)
+		atlasDBUser.Links = nil
+		operatorDBUser, err := user.ToAtlasSDK(context.Background(), k8sClient)
 		Expect(err).ToNot(HaveOccurred())
 
 		Expect(*atlasDBUser).To(Equal(normalize(*operatorDBUser, projectID)))
 	})
 }
 
 // normalize brings the operator 'user' to the user returned by Atlas that allows to perform comparison for equality
-func normalize(user mongodbatlas.DatabaseUser, projectID string) mongodbatlas.DatabaseUser {
-	if user.Scopes == nil {
-		user.Scopes = []mongodbatlas.Scope{}
-	}
-	if user.Labels == nil {
-		user.Labels = []mongodbatlas.Label{}
-	}
-	if user.LDAPAuthType == "" {
-		user.LDAPAuthType = "NONE"
-	}
-	if user.AWSIAMType == "" {
-		user.AWSIAMType = "NONE"
+func normalize(user admin.CloudDatabaseUser, projectID string) admin.CloudDatabaseUser {
+	if !user.HasScopes() {
+		user.SetScopes([]admin.UserScope{})
 	}
-	if user.X509Type == "" {
-		user.X509Type = "NONE"
+	if !user.HasLabels() {
+		user.SetLabels([]admin.ComponentLabel{})
 	}
-	if user.OIDCAuthType == "" {
-		user.OIDCAuthType = "NONE"
-	}
-	if user.DeleteAfterDate != "" {
-		user.DeleteAfterDate = timeutil.FormatISO8601(timeutil.MustParseISO8601(user.DeleteAfterDate))
-	}
-	user.GroupID = projectID
-	user.Password = ""
```

Comment:
This is not needed any more? Does the `DbUser` from the new SDK not require any normalization?